CrabZoomer.js

```diff
@@ -460,6 +496,42 @@ define([
         this._nummerFilteringSelect.set('value', value.name);
       }
       this._numberChange();
+    },
+
+    _getGemeentenByGewest: function(gewesten) {
+      // console.debug('CrabZoomer::_getGemeentenByGewest', gewesten);
+      var promises = [];
+      array.forEach(gewesten,function(gewest){
+        promises.push(this._crabGet('gewesten/' + gewest + '/gemeenten'));
+      }, this);
+      return all(promises).then(
+        lang.hitch(this, function (responses) {
+          var gemeenten = [];
+          array.forEach(responses,function(item){
```

Comment:
Is dat item dat je terugkrijgt van responses hier een array? Anders moet je daar volgens mij een .push gebruiken ipv een concat mocht het geen array zijn

Reply:
diene all() returnt steeds een array van arrays